service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CaseDataConverterTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.buildCaseDataForCMODocmosisGeneration;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.callbackRequest;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, CaseDataConverter.class})
+class CaseDataConverterTest {
+
+    @Autowired
+    private CaseDataConverter caseDataConverter;
+
+    @Test
+    void shouldConvertCaseDataToMap() {
+        assertThat(caseDataConverter.convertToMap(getCaseData())).containsKey("caseLocalAuthority");
```

Comment:
i found this a bit weird, why we checking particular properties only?
I would say that ObjectMapper should be Mock/Spy so we can assert that ObjectMapper has been  invoked with correct arguments and what is returned by this method is what we get back from object mapper. We dont want to test objectmapper

Reply:
I was not sure if tests add much value for this entire class but did half hearted attempt :). 

I have added as you suggested, do you think there is value in this now?